### ISSUE TITLE
Investigate implementing a email->FAS id schema utility

### ISSUE THREAD

**Issue by jeremycline** [fedora-infra/fedora-messaging#167]
fedmsg_meta had a function to turn an email into a FAS account (https://github.com/fedora-infra/fedmsg_meta_fedora_infrastructure/blob/develop/fedmsg_meta_fedora_infrastructure/fasshim.py).

We definitely don't want to pull that function in as-is, but we will probably want to offer that functionality. In the short term we could recommend continuing to depend on fedora_meta for that utility function, but that's not ideal because it likely pulls in a huge number of dependencies.

CC: @AdamWill

**Comment by AdamWill** [comment]
Yeah, this is pretty important for the bugzilla messages because we really can't tell "who" in Fedora terms the messages relate to any other way.
